lib/core/device.js

```diff
@@ -416,6 +441,30 @@ device.detect = function(driverId, options, cb) {
         return cb();
       }
 
+      self._deviceInfoCache[driverId] = _.cloneDeep(devdata);
+
+      self.detectHelper(driverId, options, function(err, usbDevice) {
+
```

Comment:
If we're moving towards `eslint-config-airbnb`, there shouldn't be an empty line after an opening brace here.

Reply:
`lib/core/device.js` is still ES5, so doesn't yet adhere to the new linting rules. We should probably come up with a strategy on how/if we want to migrate them over?